packages/utils/src/apis/ceos/activityApi.tsx

```diff
@@ -1,8 +1,16 @@
 import { ceosInstance } from '../axiosConfig';
 
 export const activityApi = {
-  GET_ACTIVITY: async () => {
-    const response = await ceosInstance.get(`/activities`);
+  GET_ACTIVITY: async ({
+    pageNum,
+    limit,
+  }: {
+    pageNum: number;
+    limit: number;
+  }) => {
+    const response = await ceosInstance.get(
+      `/activities?pageNum=${pageNum}&limit=${limit}`,
+    );
```

Comment:
```suggestion
const response = await ceosInstance.get(
      `/activities`,{
        params: { pageNum, limit },
      });
```
      
 params로 넣어 주면 더 가독성이 좋아질 거 같아요!!